src/main/java/subway/line/domain/Line.java

```diff
@@ -31,6 +32,27 @@ public Line(Long id, String name, String color, Sections sections) {
         this.sections = sections;
     }
 
+    public Line(String name, String color, int extraFare) {
+        this.name = name;
+        this.color = color;
+        this.extraFare = extraFare;
+    }
+
+    public Line(Long id, String name, String color, int extraFare) {
+        this.id = id;
+        this.name = name;
+        this.color = color;
+        this.extraFare = extraFare;
+    }
+
+    public Line(Long id, String name, String color, int extraFare, Sections sections) {
+        this.id = id;
+        this.name = name;
+        this.color = color;
+        this.extraFare = extraFare;
+        this.sections = sections;
+    }
```

Comment:
마찬가지로 아래와 같이 수정가능해요.
```suggestion
public Line(Long id, String name, String color, int extraFare, Sections sections) {
        this(id, name, color, extraFare);
        this.sections = sections;
    }

Reply:
생성자를 최대한 정리해보겠습니다!